x-pack/platform/plugins/shared/cases/server/saved_object_types/cases/schemas/v4.ts

```diff
@@ -14,14 +14,5 @@ export const casesSchema = casesSchemaV3.extends({
   time_to_acknowledge: schema.maybe(schema.nullable(schema.number())),
   time_to_investigate: schema.maybe(schema.nullable(schema.number())),
   time_to_resolve: schema.maybe(schema.nullable(schema.number())),
-  incremental_id: schema.maybe(
-    schema.nullable(
-      schema.oneOf([
-        schema.number(),
-        schema.object({
-          keyword: schema.maybe(schema.string()),
-        }),
-      ])
-    )
-  ),
+  incremental_id: schema.maybe(schema.nullable(schema.number())), // Added again to account for the .keyword mapping
```

Comment:
So, because you're extending v3 which has `incremential_id` you can omit `incremental_id` here. If you want to add something, maybe a comment like:

```ts
// Also updated mappings for the incremental_id field
```

Nothing bad should happen if you leave it, just not necessary to redeclare it.

Reply:
Done, thanks for the feedback!